File: BillsReact/billsApi/BillsApi/EntityFramework/Bills.cs
namespace BillsApi.DAL
{
    /// <summary>
    /// Bills table
    /// </summary>
    public class Bills
    {
        //unique ID
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int Amount { get; set; }
    }
}

File: BillsReact/bills/src/Components/bills.js
import React from 'react'

const Bills = ({ bills }) => {
  return (
    <div>
      <center><h1>Upcoming Bills</h1></center>
      {bills.map((bill) => (
        <div class="card">
          <div class="card-body">
            <h5 class="card-title">{bill.name}</h5>
            <h6 class="card-subtitle mb-2 text-muted">{bill.description}</h6>
            <p class="card-text">{bill.amount}</p>
          </div>
        </div>
      ))}
    </div>
  )
};

export default Bills